src/quest_manager/templates/quest_manager/tab_quests_available.html

```diff
@@ -7,19 +7,49 @@
     > DRAFTS (tab_quests_available.html)
     > ARCHIVED (tab_quests_available.html)
 -->
+{% if request.user.is_staff and "?bulk_edit" in request.get_full_path %}
```

Comment:
Can we move this logic into the view  e.g. `if 'bulk_edit' in request.GET ` and pass it to the template as a context variable to simplify the template logic? e.g.
`{% if bulk_edit_mode %}`  Checking for staff should be redundant since it can be handled in the view too